### PR TITLE
fix: check if document exists and fix #775

### LINKED ISSUE #775: document is not defined
**Do you want to request a *feature* or report a *bug*?**

- [ x] I found a bug
- [ ] I want to propose a feature

**What is the current behavior?**
Version 10.0.0 cause a document undefined error with server side rendering.

**If the current behavior is a bug, please provide the steps to reproduce.**

1. Enable application with SSR
1. Use react-dropzone
1.

**What is the expected behavior?**
no undefined error

**Please mention other relevant information such as the browser version, Operating System and react-dropzone version.**

Most likely caused from changing /src/utils/index.js

from
```
export const supportMultiple =
  typeof document !== 'undefined' && document && document.createElement
    ? 'multiple' in document.createElement('input')
    : true
```

to

```
export const supportMultiple = 'multiple' in document.createElement('input')
```

### PR DESCRIPTION
**Pull request by rolandjitsu**
**What kind of change does this PR introduce?**

- [x] bugfix
- [ ] feature
- [ ] refactoring / style
- [ ] build / chore
- [ ] documentation

**Did you add tests for your changes?**

- [ ] Yes, my code is well tested
- [x] Not relevant

**If relevant, did you update the documentation?**

- [ ] Yes, I've updated the documentation
- [x] Not relevant

**Summary**

Check if the `document` object exists to avoid errors in SSR.
@chill5018 I am still getting some errors after fixing this (`n` is not a function). I will need to investigate further.

**Does this PR introduce a breaking change?**

No.

**Other information**
I am still working on this PR and will try to find why the other errors occur when using SSR.

### CODE REVIEW

File: src/utils/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import accepts from 'attr-accept'
 
-export const supportMultiple = 'multiple' in document.createElement('input')
```
Comment (KaiHotz) [inline src/utils/index.js]: Wouldn't  it be enough  to just change 

`document.createElement('input')`

 to

`React.createElement('input')`  ?

Comment (sstur) [reply src/utils/index.js]: No, a React element won't really tell you what the browser is capable of, it's just a plain object afterall. The result of `React.createElement('input')` will look like this:
```js
{
  $$typeof: Symbol(react.element)
  props: {}
  type: "input"
}
```

Comment (KaiHotz) [reply src/utils/index.js]: Ok thanks for the reply

**Comment by rolandjitsu** [comment]
@okonet just tested if the fix works and there seems to be no more problems. Let me know if it's ok to merge.

P.S. I think we should eventually add some integration tests to ensure this won't break in the future.

**Comment by okonet** [comment]
Please take a lead on this project. I’m grounded with kids and other projects atm :)

**Comment by rolandjitsu** [comment]
@okonet sure thing. Good luck 👍

**Pull request merged**